index.js

```diff
@@ -138,7 +143,14 @@ function generateAttributes(attributes = {}) {
 
 	return (
 		' ' +
-		attributes.map(attribute => `${attribute[0]}="${attribute[1]}"`).join(' ')
+		attributes
+			.map(attr => {
+				if (typeof attr[1] === 'string' && attr[1].length === 0) {
```

Comment:
Probably `true` instead of `''` would be a better API. Empty string is a valid value for some attributes.